app/models/college.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+#
+# Model for site configuration / individual colleges. Will be used for
+# multi-tenancy in the future.
+#
+# @attr name [String] the college name
+# @attr admin_email [String] the admin e-mail for the college
+# @attr dean [String] the name of the college dean / head
+# @attr site_url [String] the full url (including http/https) for the Vesta site
+#   for the college
```

Comment:
I figured we'd add this stuff in the other issue, this was more to just replace what we already had.

I also don't know that we need to store housing rep e-mails separately - we'll know who they are by their user accounts and roles. In fact, we can just create `Draw#reps` and use that to display the e-mail address in the draw views.